frontend/package.json

```diff
@@ -107,6 +107,7 @@
     "semver": "^6.0.0",
     "showdown": "1.8.x",
     "text-encoding": "0.x",
+    "ts-get": "^1.0.5",
```

Comment:
This seems useful, although I'm a little hesitant to rely on a project with so little history and only one contributor.

@christianvogt @vojtechszocs opinion?

Reply:
I'd have to play around with it to see if it's more useful than `_.get(...) as SomeType`, which is what we tend to use now.

Reply:
Yes, that project is quite new but it is also extremely small - just one wrapper function (https://github.com/RIP21/ts-get/blob/master/src/index.ts). So it would be easy to pull it into our codebase if it goes under.

Reply:
I dislike using `_.get` because it's easy to miss when refactoring. Sure casting the result satisfies typescript but it's a crutch.

My first hunch regarding `ts-get` was that it used try-catch which is going to be slow. Sure enough they have a closed issue of performance:
https://github.com/RIP21/ts-get/issues/3

Reply:
> My first hunch regarding `ts-get` was that it used try-catch which is going to be slow. Sure enough they have a closed issue of performance:
> [RIP21/ts-get#3](https://github.com/RIP21/ts-get/issues/3)

Oh, wow, that is quite a difference